match/lib/match/utils.rb

```diff
@@ -9,6 +9,53 @@ def self.import(item_path, keychain)
       Helper.backticks(command, print: $verbose)
     end
 
+    # logs public key's  name, user, organisation, country, availability dates
+    def self.log_certificate_public_key(cer_certificate_path)
+      command = "openssl x509 -inform der -in #{cer_certificate_path.shellescape} -subject -dates -noout"
+      command << " &" # start in separate process
```

Comment:
Sorry, I do not understand you. What I need to change? Provision profiles are logged on every match's call. I write logging as that.